export_to_godot_tilemap.js

```diff
@@ -130,7 +138,12 @@ class GodotTilemapExporter {
                         this.extResourceId = this.extResourceId + 1;
                         textureResourceId = this.extResourceId;
                         this.tilesetsIndex.set(tilesetsIndexKey, this.extResourceId);
-                        let tilesetPath = object.tile.tileset.image.replace(this.projectRoot, "");
+                        let tilesetPath = object.tile.tileset.image;
+                        let slashIndex = object.tile.tileset.image.lastIndexOf("/");
+                        if(slashIndex >= 0) {
+                            let filePath = object.tile.tileset.image.substring(0, slashIndex);
+                            tilesetPath = object.tile.tileset.image.replace(filePath, this.projectRoot);
```

Comment:
```suggestion
                        let slashIndex = tilesetPath.lastIndexOf("/");
                        if(slashIndex >= 0) {
                            let filePath = tilesetPath.substring(0, slashIndex);
                            tilesetPath = tilesetPath.replace(filePath, this.projectRoot);
```
Though again, it seems to me that this should rather be something like:
```js
let tilesetPath = FileInfo.joinPaths(this.projectRoot, FileInfo.fileName(tilesetPath));
```